src/app/(admin)/clients/[id]/staff/_components/StaffFormModal.tsx

```
@@ -0,0 +1,135 @@
+    const { toast } = useToast();
+
+    const handleSubmit = async (data: StaffFormValues) => {
+        try {
+            // 1. Check if user exists by email
+            const email = data.email as string; // Type assertion since we know it's a string from the Zod schema
+            const userCheckRes = await fetch(`/api/users?email=${encodeURIComponent(email)}`);
+            let user, profileId;
+
+            if (userCheckRes.ok) {
+                const existing = await userCheckRes.json();
+                if (existing && existing.id && existing.profileId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)